src/shared/extensions.ts

```diff
@@ -14,6 +14,8 @@ import { CredentialsStore } from '../credentials/credentialsStore'
 export const VSCODE_EXTENSION_ID = {
     awstoolkit: 'amazonwebservices.aws-toolkit-vscode',
     python: 'ms-python.python',
+    // python depends on jupyter plugin
+    jupyter: 'ms-toolsai.jupyter',
```

Comment:
just curious (also maybe good to add to the comment), but what part of python functionality depends on jupyter? was not needed for existing integration tests...